diff --git a/node_modules/@capacitor/android/capacitor/proguard-rules.pro b/node_modules/@capacitor/android/capacitor/proguard-rules.pro
index 96db065..8a9971c 100644
--- a/node_modules/@capacitor/android/capacitor/proguard-rules.pro
+++ b/node_modules/@capacitor/android/capacitor/proguard-rules.pro
@@ -6,18 +6,18 @@
 #   http://developer.android.com/guide/developing/tools/proguard.html
 
 # Rules for Capacitor v3 plugins and annotations
- -keep @com.getcapacitor.annotation.CapacitorPlugin public class * {
+ -keep,allowoptimization @com.getcapacitor.annotation.CapacitorPlugin public class * {
+     public <init>();
      @com.getcapacitor.annotation.PermissionCallback <methods>;
      @com.getcapacitor.annotation.ActivityCallback <methods>;
      @com.getcapacitor.annotation.Permission <methods>;
      @com.getcapacitor.PluginMethod public <methods>;
  }
 
- -keep public class * extends com.getcapacitor.Plugin { *; }
-
 # Rules for Capacitor v2 plugins and annotations
 # These are deprecated but can still be used with Capacitor for now
--keep @com.getcapacitor.NativePlugin public class * {
+-keep,allowoptimization @com.getcapacitor.NativePlugin public class * {
+  public <init>();
   @com.getcapacitor.PluginMethod public <methods>;
 }
 
